packages/mirinae/src/inputs/buttons/toggle-button/config.ts

```diff
@@ -1,6 +1,20 @@
-export const TOGGLE_BUTTON_THEME = Object.freeze({
+export const TOGGLE_BUTTON_THEME = {
     secondary: 'secondary',
     peacock500: 'peacock500',
-} as const);
+} as const;
+
+export const TOGGLE_BUTTON_SPACING = {
+    none: 'none',
+    sm: 'sm',
+    md: 'md',
+    lg: 'lg',
+    spaceBetween: 'space-between',
+} as const;
+
+export const TOGGLE_BUTTON_POSITION = {
+    right: 'right',
+    left: 'left',
+    top: 'top',
+} as const;
```

Comment:
praise: 👍

Reply:
suggestion: It can be used as type in PToggleButton.vue!

```typescript
export type ToggleButtonSpacing = keyof typeof TOGGLE_BUTTON_SPACING;

export type ToggleButtonPosition = typeof TOGGLE_BUTTON_POSITION[keyof typeof TOGGLE_BUTTON_POSITION];
```